src/BalconyBotanica/Core/DomainObjects/QuizQuestions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BalconyBotanica.Core.DomainObjects
+{
+    public class QuizQuestions
+    {
+        public Sunlight sunlight;
+        public int spaceSize;
```

Comment:
Right now it isn't possible for the reader to know what the units of `spaceSize` is. It could be a number where 1 is small and 10 is large or it could be square meters or square feet (remember the internet is global ;-) ). Good practice is to declare a type for this or to keep things simple in this toy project, put the units in the name of the variable. For example, `spaceSizeSquareMeters`